src/pages/Trade/components/RadioButtons/RadioButtons.tsx

```diff
@@ -1,14 +1,45 @@
 import { Positions } from 'enums/options';
-import React from 'react';
+import useMarketsCountQuery from 'queries/options/useMarketsCountQuery';
+import React, { useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { getIsAppReady } from 'redux/modules/app';
+import { getNetworkId } from 'redux/modules/wallet';
 import styled from 'styled-components';
+import { RootState } from 'types/ui';
+import { areDatesEqual } from 'utils/ui';
 
 type RadioButtonsProps = {
     selected: Positions;
     onChange: React.Dispatch<Positions>;
     options?: Positions[];
+    currencyKey?: string;
+    date?: number | undefined;
```

Comment:
type can be only number as question mark is already marking the property as optional(undefined)